homeworks/vladimir.karbivnychiy_VladimirSkarb/homework_7/fibo.test.js

```diff
@@ -0,0 +1,11 @@
+const { fiboNumb } = require('./fibo');
+
+test('test fibonacci', () => {
+    const minFibonacciArray = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34];
+    expect(fiboNumb(50)).toEqual(minFibonacciArray);
+});
+
+test('test fibonacci', () => {
+    const minFibonacciArray = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55];
+    expect(fiboNumb(55)).toEqual(minFibonacciArray);
+});
```

Comment:
test should have informative description, e.g. 'first element of array should be 0'

Reply:
also, better to use loop to test each element to follow Fibo sequence